### PR TITLE
Writing the README.md

### LINKED ISSUE #9: DEV: Create an appropriate README.md
The boilerplate README.md file needs to be updated. The updated version of the README.md file should (at the very minimum) contain headers answering the following questions:
-  What is the new cuBlueprint site? What's on it?
- How to run it locally?
- What was used to create this project?

### PR DESCRIPTION
**Pull request by Seneli**
Hello I'm still working on this but give it a read over and let me know your feedback for the format I have going so far. I can write any added sections or descriptions you want to see :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,44 @@
...
+Our new and updated website for CUBluePrint!
 
-## Available Scripts
+You can find our live website available at: [link](enter link here thanks)
+
+
+
+## Motivation for Project
+
+In 2021, we decided to build our club a brand new website with an improved UI and a more modern tech stack. The reason for this change was that our old website was built with __[insert outdated technology]__ and was becoming increasingly difficult for our exec team to maintain.
```
Comment (deenhaque) [inline README.md]: I'm pretty sure the old [site](https://github.com/cublueprint/cublueprint-website) used Pug, a templating engine taught in COMP2406. I think the main motivation for the revamp was just the outdated design decisions.

File: README.md [hunk 2]
```diff
@@ -1,18 +1,44 @@
...
+
+To meet these challenges, our team designed a [new UI using Figma](https://www.figma.com/file/Qt7bGynNItu0hUvPu0EPGa/Draft-Website-Design?node-id=0%3A1) and is currently working on developing it using React, Typescript and styled components.
```
Comment (deenhaque) [inline README.md:13]: can we have the 💅 after styled LOL

File: README.md [hunk 3]
```diff
@@ -29,18 +55,11 @@ Your app is ready to be deployed!
...
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## Credits
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Credits for the website go to:
+* Deen
```
Comment (deenhaque) [inline README.md]: Should the names be links to stuff?

if @meerabal and @samee-shahood are feeling it, im down to have my name link to www.linkedin.com/in/deenhaque

File: README.md [hunk 4]
```diff
@@ -1,18 +1,44 @@
...
+* React
+* Yarn
+* Styled Components
+
+
+## How to run the website in Development
+
+To run the project in development mode on a local server, run the following command inside the project directory:
+
+### `yarn start`
```
Comment (deenhaque) [inline README.md:39]: I have to admit, I'm surprised by how good this looks. 👍

**Comment by deenhaque** [comment]
README looks good! I just left some basic feedback to address

File: README.md (hunk 4 above)
Comment (Seneli) [reply README.md:39]: ratio'd on my own PR <3

Comment (deenhaque) [reply README.md:39]: LOL

File: README.md [hunk 5]
```diff
@@ -29,18 +55,11 @@ Your app is ready to be deployed!
...
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## Credits
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+Credits for the website go to:
+* [Deen](www.linkedin.com/in/deenhaque)
```
Comment (deenhaque) [inline README.md]: i think this needs an https before, LOL

You can try actually clicking on it in the .md file to see what happens
Sorry for giving you a bad link 😅

File: README.md [hunk 6]
```diff
@@ -29,18 +55,11 @@ Your app is ready to be deployed!
...
+* [Meerah](https://www.linkedin.com/in/meera-balsara/)
```
Comment (meerabal) [inline README.md]: Sorry for nitpicking but can you change the spelling to Meera without the h? :')

Comment (deenhaque) [reply README.md]: LMAO

Comment (Seneli) [reply README.md]: done aoushdoahsuda I'm so sorry again yikess

File: README.md (hunk 5 above)
Comment (Seneli) [reply README.md]: done

**Review by deenhaque — APPROVED** [review]
LGTM

**Pull request merged**